pipeline/jobs/unittest5_job.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+SCRIPT_DIR="$( cd "$(dirname "$0")" ; pwd -P )"
+. "${SCRIPT_DIR}/params.sh"
+
+${ROOT_DIR}/pipeline/stage_unittests/trigger.sh \
+${FORWARD_OPTS[@]} \
+--sourcedir $BUILD_DIR \
+--targetdir $TEST_DIR \
+--testcategories Filter,Targeted,Purge,Serialization,WebApi,Utility,UriUtility,IO,ImportExport,Disruption | ts
```

Comment:
It would be nice if we didn't have to remember to add an entry here every time we add a new test category.  Is there any way for this to be a "catch-all" by specifying which categories to exclude instead?  That way, we can say run all test categories except the ones in the other jobs.

Reply:
I moved them to the travis yaml and local start.sh script. Maybe later in another PR we can have such a mechanism that can do that, because you're right, it is a bit redundant to specify them again here unless we want to dedicate a travis worker to it.